operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml

```diff
@@ -111,7 +111,7 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:label
       version: v1alpha1
-    - description: Represents an installation of latest version of Addons (exclusively Tasks, StepActions, consoleyamlsamples, consoleclidownloads and consolequickstarts)
+    - description: Represents an installation of latest version of Addons (exclusively ClusterTasks, consoleyamlsamples, consoleclidownloads and consolequickstarts)
```

Comment:
@pratap0007 you need to update your changes on the CSV template file,
see our template tool is replacing your changes.

https://github.com/tektoncd/operator/pull/2376/files#diff-09c721bf5f37ab86c75b41674b7ef75349b4b30971e7d67e595d92fe602ed157R114